src/components/Layout/ContentHeader.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import Icon from '/@/components/UI/Icon.vue'
+
+interface Props {
+  iconName: string
+  title: Array<string>
+  detail: string
+}
+const props = defineProps<Props>()
+
+const titleList = props.title.map((text, index) => ({ id: index, text: text }))
+</script>
+
+<template>
+  <div :class="$style.container">
+    <div :class="$style.titleContainer">
+      <icon :class="$style.icon" :name="props.iconName" :size="48" />
+      <template v-for="{ id, text } in titleList" :key="id">
+        <Icon
+          v-if="id !== 0"
+          :class="$style.chevron"
+          name="akar-icons:chevron-right"
+          :size="30"
+        />
+        <h1 :class="$style.title">{{ text }}</h1>
+      </template>
+    </div>
```

Comment:
全体を大きくh1タグで囲っちゃっていい気がしたのですがどうですか
```suggestion
    <h1 :class="$style.titleContainer">
      <icon :class="$style.icon" :name="props.iconName" :size="48" />
      <template v-for="{ id, text } in titleList" :key="id">
        <Icon
          v-if="id !== 0"
          :class="$style.chevron"
          name="akar-icons:chevron-right"
          :size="30"
        />
        <p :class="$style.title">{{ text }}</p>
      </template>
    </h1>
```

Reply:
https://momdo.github.io/html/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements
↑によると hタグのコンテンツモデルはフレージングになっています。 Iconコンポーネントで、 div 要素を使用しているので、やや微妙な感じがしますね。。。

Reply:
なるほどです、確かにそうなると微妙ですね...

Reply:
そもそもIconコンポーネントでdivを使わなくてもいい説があったりなかったりします？